### DISCUSSION TITLE
Yi Home - RFUSY6231D - need help

### DISCUSSION THREAD

**Discussion by simonricher007** [roleoroleo/yi-hack-Allwinner-v2#500] [Q&A]
Hi! 
I'm looking for the right file for my camera. 
I can't find my model listed.

Yi Home - RFUSY6231D
Firmware: 11.1.35.202207191613

Thanks for your help.

**Comment by roleoroleo** [comment]
Your cam is currently not supported.
Please make a backup copy of the flash and share it.
I will check if I can support it.
